tutorials/llama_index_search_and_retrieval_tutorial.ipynb

```diff
@@ -0,0 +1,819 @@
+{
```

Comment:
<p>Emoji is tad confusing. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/891/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Changed to 💬</p>